packages/web-components/src/components/background-media/background-media.scss

```
@@ -21,6 +21,13 @@
   @include tooltip--placement('definition', 'top', 'end');
 
   position: absolute;
+  border: 2px solid transparent;
```

A bit of nitpicking, but let's use the `$spacing-01` token which renders as `2px` or `0.125rem`.

```suggestion
  border: $spacing-01 solid transparent;
```